packages/react-native/React/CoreModules/RCTPerfMonitor.mm

```diff
@@ -183,7 +183,9 @@ - (UIPanGestureRecognizer *)gestureRecognizer
 - (UIView *)container
 {
   if (!_container) {
-    _container = [[UIView alloc] initWithFrame:CGRectMake(10, 25, 180, RCTPerfMonitorBarHeight)];
+    CGSize statusBarSize = [UIApplication sharedApplication].statusBarFrame.size;
```

Comment:
Causes the bellow error internally
> 'sharedApplication' is unavailable: not available on iOS (App Extension) - Use view controller based solutions where appropriate instead.

In RCTStatusBarManager.mm I see usage of `RCTSharedApplication().statusBarFrame.size.height`

Reply:
Hi nick, I'm basically a newbie in iOS dev. Please kindly check if it is right to change `[UIApplication sharedApplication]` to `[RCTSharedApplication()]`. I've pushed the modification